portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/ExportAPI.testcase

```diff
@@ -157,6 +139,46 @@ definition {
 		}
 	}
 
+	@description = "Feature ID: LRAC-12705 | Automation ID: LRAC-13592 | Test Summary: An internal problem occurred during the export process"
+	@priority = 2
+	test InternalProblemOccurredInExportProcess {
+		var startDate = "1987-05-13";
+		var startTime = "11:45:00.000";
+		var endDate = "2008-05-13";
+		var endTime = "09:20:00.000";
+		var type = "page";
+		var message = "An internal problem happened when trying to reach our services";
+		var containerName = "osbasahbackend";
+
+		task ("Get the token value") {
+			var token = ACApi.getTokenValue();
+		}
+
+		task ("Stop the osbasahbackend container") {
+			var containerID = ExecUtil.executeCommand("docker ps -aqf name=${containerName}");
+
+			ExecUtil.executeCommand("docker stop ${containerID}");
+		}
+
+		task ("Create a data export using analytics cloud API with invalid type") {
+			ACApi.createNewExportSchedule(
+				endDate = ${endDate},
+				endTime = ${endTime},
+				startDate = ${startDate},
+				startTime = ${startTime},
+				token = ${token},
+				type = ${type});
+		}
+
+		task ("Check if the API response is correct for the error case") {
+			ACApi.viewApiResponseForError(message = ${message});
+		}
+
+		task ("Start the osbasahbackend container") {
+			ExecUtil.executeCommand("docker start ${containerID}");
+		}
```

Comment:
Is it possible to add this somewhere in the teardown in case this test breaks any time after we're stopping the container? Since if this test fails it might cause other tests to fail without the container being up.

Reply:
@ViniLopes20

Reply:
Hey @joshitagaki, good observation, I think we can put it in `tearDown` and even for cases where the container is already running, it won't affect running the `docket start` again. Thanks

Reply:
Thanks everyone!